Empty databases/openIMIS_ONLINE.sql

```diff
@@ -9953,6 +9953,62 @@ CREATE PROCEDURE [dbo].[uspConsumeEnrollments](
 			AND TP.isOffline = 0
 			
 
+            /********************************************************************************************************
+                                        CHECK IF SOME INSUREE ARE ABOUT TO DELETE FROM FAMILY	
+            ********************************************************************************************************/
+
+            -- get the family id to process from online database
+            DECLARE @familyIdToProcess TABLE (FamilyId INT)
+            INSERT INTO @familyIdToProcess(FamilyId)
+            SELECT I.FamilyId
+            FROM @tblInsuree TI
+            INNER JOIN tblInsuree I ON TI.CHFID = I.CHFID
+            WHERE I.ValidityTo IS NULL AND TI.isOffline = 1
+            GROUP BY I.FamilyID
+
+            -- get to compare the structure of families (list of insuree) from online database
+            DECLARE @insureeToProcess TABLE(CHFID NVARCHAR(12), FamilyID INT) 
+            INSERT INTO @insureeToProcess(CHFID, FamilyID)
+            SELECT I.CHFID, F.FamilyID FROM tblInsuree I 
+            LEFT JOIN tblFamilies F ON I.FamilyID = F.FamilyID
+            WHERE F.FamilyID IN (SELECT * FROM @familyIdToProcess) AND I.ValidityTo is NULL
+            GROUP BY I.CHFID, F.FamilyID
+
+            -- select the insuree to delete based on received XML payload
+            -- get the insuree which are not included in "insureeToProcess"
+            DECLARE @insureeToDelete TABLE(CHFID NVARCHAR(12))
+            INSERT INTO @insureeToDelete(CHFID)
+            SELECT CHFID
+            FROM (
+            SELECT CHFID FROM @insureeToProcess
+            UNION ALL
+            SELECT CHFID FROM @tblInsuree TI
+            ) tbl
+            GROUP BY CHFID
+            HAVING count(*) = 1
```

Comment:
This will get also the new insuree

Why not doing a left join 

SELECT CHFID FROM @insureeToProcess IP
LEFT JOIN @tblInsuree I ON I.CHFID=IP.CHFID  
WHERE I.CHFID is NULL

Reply:
Ok @delcroip . I will try to do this and check if it works. 
Thx for clue.